### DISCUSSION TITLE
How to add "alt" to the description of photos?

### DISCUSSION THREAD

**Discussion by sztabas** [datenstrom/community#437] [Ask a question] [answered]
I'm sorry, I still have a question: how to add alt to the description of photos, please give an example. Thank you

**Comment by hysonlee** [comment, marked as answer]
For example if you're using official plugin:`[image filename-at-default-folder.png "type your photo description here"]` ,
Ps. The `README.md` file is a good friend.

**Comment by sztabas** [comment]
Thank you so much, it helped me a lot.

niedz., 14 lip 2019 o 05:06 Hyson Lee <notifications@github.com> napisał(a):

> For example if you're using official plugin:[image
> filename-at-default-folder.png "type your photo description here"] ,
> Ps. The README.md file is a good friend.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/datenstrom/yellow/issues/437?email_source=notifications&email_token=AMRR32CRUYBYBGFK6MUUOSDP7KJ45A5CNFSM4H7N6BP2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZ35GKY#issuecomment-511169323>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMRR32E2MWIFTMKCMIAZRVDP7KJ45ANCNFSM4H7N6BPQ>
> .
>


-- 
Pozdrawiam
-----------------------------
*Dariusz Sztabler*
